src/components/Calculator/operate.js

```diff
@@ -0,0 +1,26 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+    return one.mod(two).toString();
+  }
+  throw Error(`Unknown operation '${operation}'`);
+}
```

Comment:
- [OPTIONAL] It would be better if your modulus operation works in the same way as your division operation works with zero.  `Can't divide by 0.` because when I try to use the modulus with a number and zero. It throws an error.

![modulus](https://user-images.githubusercontent.com/79876959/144521148-a5b73b98-00b4-4462-8698-679cbf403dc2.PNG)

Reply:
Thanks! I just fixed this, now works even better.